### PR TITLE
ID localization for configure pod and container - assign memory resource

### PR DESCRIPTION
**Pull request by ariscahyadi**
This PR translates `/docs/tasks/configure-pod-container/assign-memory-resource/` as mentioned here #22296

### CODE REVIEW

File: content/id/docs/tasks/configure-pod-container/assign-memory-resource.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+kubectl delete pod memory-demo --namespace=mem-example
+```
+
+## Melebihi batasan memori dari Container
+
+Container dapat melebihi permintaan memorinya jika Node memiliki memori yang tersedia. Tapi sebuah Container
+tidak diperbolehkan untuk menggunakan lebih dari batasan memorinya. Jika Container mengalokasikan lebih banyak memori daripada
+batasannya, Container menjadi kandidat untuk dihentikan. Jika Container terus berlanjut
+mengkonsumsi memori melebihi batasnya, maka Container akan diakhiri. Jika Container dihentikan dan bisa
+di mulai ulang, kubelet akan memulai ulang, sama seperti jenis kegagalan _runtime_ yang lainnya.
```
Comment (girikuncoro) [inline content/id/docs/tasks/configure-pod-container/assign-memory-resource.md:136]: nit:
```suggestion
dimulai ulang, kubelet akan memulai ulang, sama seperti jenis kegagalan _runtime_ yang lainnya.
```

File: content/id/docs/tasks/configure-pod-container/assign-memory-resource.md [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+...
+```
+
+Jalankan `kubectl top` untuk mengambil metrik dari Pod:
+
+```shell
+kubectl top pod memory-demo --namespace=mem-example
+```
+
+Keluarannya menunjukkan bahwa Pod menggunakan memori sekitar 162.900.000 byte, dimana
```
Comment (girikuncoro) [inline content/id/docs/tasks/configure-pod-container/assign-memory-resource.md:114]: nit: italic
```suggestion
Keluarannya menunjukkan bahwa Pod menggunakan memori sekitar 162.900.000 _byte_, dimana
```

**Review by girikuncoro — APPROVED** [review]
This is beautifully translated, thanks for your continuous contribution! 🎉 
I left very nit comments, can go to follow up PR. This can get in now.

**Comment by ariscahyadi** [comment]
Thanks @girikuncoro for direct approved ... I will address some nits next time.

**Pull request merged**